pom.xml

```diff
@@ -87,7 +87,7 @@ THE SOFTWARE.
     <patch.tracker.serverId>jenkins-jira</patch.tracker.serverId>
 
     <guavaVersion>11.0.1</guavaVersion>
-    <slf4jVersion>1.7.7</slf4jVersion> <!-- < 1.6.x version didn't specify the license (MIT) -->
+    <slf4jVersion>1.7.25</slf4jVersion>
```

Comment:
Yes the intent here (together with the `plugin-pom` PR) was to allow a plugin using “newest everything” to build without warnings.